src/lib/circular-progress/circular-progress.ts

```diff
@@ -21,20 +20,41 @@ declare global {
 }
 
 /**
- * The web component class behind the `<forge-circular-progress>` custom element.
- * 
  * @tag forge-circular-progress
+ * 
+ * @summary
+ * Circular progress indicators display progress by animating along a circular track in a clockwise direction.
+ * 
+ * @description
+ * Progress indicators inform users about the status of ongoing processes.
+ * - Determinate indicators display how long a process will take.
+ * - Indeterminate indicators express an unspecified amount of wait time.
+ *
+ * @property {boolean} determinate - Controls the determinate state.
+ * @property {boolean} progress - Controls the progress while in a determinate state. Accepts values from `0` to `1`.
+ * 
+ * @attribute {boolean} determinate - Controls the determinate state.
+ * @attribute {string} data-aria-label - Propagates an `aria-label` to the underlying progressbar element.
+ * 
+ * @slot - The is the default/unnamed slot. Renders content at the center of the progress indicator.
+ * 
+ * @cssproperty --forge-circular-progress-size - The height and width of the indicator container.
+ * @cssproperty --forge-circular-progress-padding - The padding inside the bounding box of the container.
+ * @cssproperty --forge-circular-progress-indicator-width - The active indicator width.
+ * @cssproperty --forge-circular-progress-indicator-color - The active indicator color.
+ * @cssproperty --forge-circular-progress-indicator-background-color - The indicator background color.
+ * 
+ * @csspart progressbar - Styles the progress bar container element
```

Comment:
Nice to see all these custom manifest attributes being specified!  Definitely would be nice to get the corresponding portions of the docs generated.

Reply:
I plan to do a POC of generating the docs from the manifest pretty soon. I'll follow up with you on any findings when I get there.